geometry/polygon_area.cpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file
+ * @brief Implementation of [Shoelace
+ * formula](https://en.wikipedia.org/wiki/Shoelace_formula).
```

Comment:
Is this even the implementation of the Shoelace Formula? 😅
```suggestion
 * @brief Implementation of [Polygon Area](https://en.wikipedia.org/wiki/https://en.wikipedia.org/wiki/Polygon).
```